raiden-dapp/src/components/dialogs/NoChannelsDialog.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <raiden-dialog class="no-channels-dialog" :visible="visible" hide-close>
+    <v-card-title>{{ $t('transfer.no-channels-dialog.title') }}</v-card-title>
+    <div>
+      <v-img
+        class="no-channels-dialog__warning"
+        :src="require('@/assets/warning.svg')"
+        height="132px"
+        width="150px"
+      />
+      <action-button
+        :text="$t('transfer.no-channels-dialog.button')"
+        enabled
+        @click="navigateToTokenSelect()"
+      />
+    </div>
+  </raiden-dialog>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Mixins } from 'vue-property-decorator';
+import RaidenDialog from '@/components/dialogs/RaidenDialog.vue';
+import ActionButton from '@/components/ActionButton.vue';
+import NavigationMixin from '../../mixins/navigation-mixin';
+
+@Component({
+  components: { RaidenDialog, ActionButton },
+})
+export default class NoChannelsDialog extends Mixins(NavigationMixin) {
+  @Prop({ required: true })
+  visible!: boolean;
```

Comment:
Maybe I lack in knowledge about the dialog, but I would rather put the render condition where used in the parent component and not here with a prop. Else I think we could end up in an awkward  mix of different pattern.

Reply:
Ignore this comment. I think this is perfectly fine.